src/app/components/OptimoPromos/MediaIndicator/index.styles.jsx

```diff
@@ -0,0 +1,13 @@
+import { GEL_SPACING } from '#app/legacy/gel-foundations/src/spacings';
+import styled from '@emotion/styled';
+
+export const MediaIndicatorWrapper = styled.div`
+  display: inline-block;
+  padding-right: ${GEL_SPACING};
+`;
+
+export const MediaIndicatorFlexWrapper = styled.div`
```

Comment:
from the standards `Avoid using visually descriptive names for components.`

Reply:
Renamed to MediaIndicatorAlignment since it is the style that deals with aligning the indicator correctly.

Reply:
Personally I'd have went with `MediaIndicatorWrapper`. Wrapper is quite a common name for a component and the alignment is a visual thing

Reply:
I already have a MediaIndicatorWrapper, something like the following:
```
<MediaIndicatorWrapper>
  <MediaIndicatorAlignment>
   {The media indicator svg}
  </MediaIndicatorAlignment>
</MediaIndicatorWrapper>
```

The MediaIndicator Wrapper takes care of the inline styling with the heading, and the MediaIndicatorAlignment takes care of the vertical alignment.
This is how we currently do our media indicators, so I took inspiration from that.